addon/templates/components/polaris-progress-bar.hbs

```diff
@@ -1,10 +1,12 @@
-<progress class="Polaris-ProgressBar__Progress"
+<progress
+  class="Polaris-ProgressBar__Progress"
   max="100"
-  value="{{parsedProgress}}"
+  value={{parsedProgress}}
```

Comment:
Could this removal of the quotes cause issues with host apps that pass numeric values instead of strings?

Reply:
I don't think they should, but good point....will double check